.github/renovate.json5

```diff
@@ -6,18 +6,20 @@
   ],
   "labels": ["dependencies"],
   "ignorePaths": [],
-  "golang": {
-    "enabled": false,
-  },
-  "github-actions": {
-    "enabled": false,
-  },
   "packageRules": [
     {
       "matchPackageNames": [
         "ghcr.io/aquasecurity/trivy",
       ],
       "semanticCommitType": "build",
     },
+    {
+      "matchManagers": ["gomod"],
+      "semanticCommitType": "build",
+    },
```

Comment:
```suggestion
```

I think we decided to do things as close to default for now. When introducing the reusable release-please workflow this will change anyway.

Reply:
I am working on the workflow changes PR.

Reply:
I couldn't find a reference to that, so I just implemented it 1-1, but happy to remove. I think you are missing a line in your suggestion, pushing a fix.